### PR TITLE
feat(docs): update Callout and Alert guidelines

### PR DESCRIPTION
**Pull request by serifluous**
Pages with content updates:

- [Notifications and feedback](https://paste-docs-git-alert-callout-guideline-update-twilio.vercel.app/patterns/notifications-and-feedback)
- [Create pattern](https://paste-docs-git-alert-callout-guideline-update-twilio.vercel.app/patterns/create#anchor): Image only
- [Alert](https://paste-docs-git-alert-callout-guideline-update-twilio.vercel.app/components/alert)
- [Callout](https://paste-docs-git-alert-callout-guideline-update-twilio.vercel.app/components/callout)
- [Page Header](https://paste-docs-git-alert-callout-guideline-update-twilio.vercel.app/components/page-header)

### CODE REVIEW

**Review by krisantrobus — DISMISSED** [review]
LGTM ⭐

File: packages/paste-website/src/assets/images/patterns/notifications-alert@3x.png
Comment (roxanagomez) [inline packages/paste-website/src/assets/images/patterns/notifications-alert@3x.png:1]: **suggestion:** update Toas image: I didn't see if you updated this image too, but I think the Toast here is an old verison

Screenshot 2024-09-11 at 4 38 54 PM

**Review by roxanagomez — COMMENTED** [review]
nitpick: In the Notification and feedback patterns the ingredients section last 2 cards have no space in between, I think since one card was added it kind of messed up the composition. 
Screenshot 2024-09-11 at 4 46 12 PM

File: packages/paste-website/src/assets/images/patterns/notifications-alert@3x.png
Comment (serifluous) [reply packages/paste-website/src/assets/images/patterns/notifications-alert@3x.png:1]: yeah, I didn't bother updating the other ones yet. During our next Think Week I'm planning on going through some of our old docs and updating them, maybe finally putting together coded examples for the older patterns.

**Review by roxanagomez — APPROVED** [review]

**Review by krisantrobus — APPROVED** [review]

**Pull request merged**